src/tools/CMakeLists.txt

```diff
@@ -0,0 +1,19 @@
+cmake_minimum_required(VERSION 3.16)
+project(host-tools)
+
+# Require standard C++14
+set(CMAKE_CXX_STANDARD 14)
```

Comment:
...don't we actually only require C++11?

Reply:
IIRC when we went through and looked at our list of supported compilers (GCC 5+, Clang 9+, MSVC 2017+), C++14 was the GCD among them.